### PR TITLE
[INLONG-988][Doc] Add cluster management document

### LINKED ISSUE #988: [Improve][Doc] Add cluster management document
### Description

Add cluster management document.

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
Fixes #988

### Motivation

Add cluster management document.

### Modifications

Add cluster management document.

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/administration/cluster_management.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: 集群管理
+sidebar_position: 5
+---
+
+角色为系统管理员或者租户管理员的用户才可以使用此功能，他们可以创建、修改、删除集群：
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/administration/cluster_management.md:6]:
```suggestion
角色为系统管理员或者租户管理员的用户才可以使用此功能，他们可以创建、修改、删除集群。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/administration/cluster_management.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+具有系统管理员或者租户管理员权限用户，可以进行创建新集群：
+
+![](img/create_cluster.png)
+
+- 集群名称：用户自定义名，用来标识此集群信息
+- 类型：集群类型
+- 集群标签：该集群所属集群标签
+- 责任人：该集群负责人，只有责任人能够修改该集群的配置信息
+- 集群描述：集群描述信息
+-
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/administration/cluster_management.md:19]:
```suggestion
```

File: docs/administration/cluster_management.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Cluster Management
+sidebar_position: 5
+---
+
+Only system administrators or tenant administrators have permission to use this feature, and they can create, modify, and delete clusters:
```
Comment (aloyszhang) [inline docs/administration/cluster_management.md:6]:
```suggestion
Only system administrators or tenant administrators have permission to use this feature, and they can create, modify, and delete clusters.
```

File: docs/administration/cluster_management.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Users with system administrator or tenant administrator permissions can create new clusters. When creating, you need to fill in the following information:
+
+![](img/create_cluster.png)
+
+- Cluster Name：A user-defined name used to identify the cluster.
+- Type：Cluster type.
+- Cluster Tag：The cluster belongs to the cluster tag
+- Owners：Specify the person responsible for the cluster, who is the only one able to modify the cluster's configuration information.
+- Description：Provide a brief description of the cluster.
+-
```
Comment (aloyszhang) [inline docs/administration/cluster_management.md:20]:
```suggestion
```

Comment (fuweng11) [reply docs/administration/cluster_management.md]: Fixed.

File: docs/administration/cluster_management.md (hunk 3 above)
Comment (fuweng11) [reply docs/administration/cluster_management.md]: Fixed.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/administration/cluster_management.md (hunk 2 above)
Comment (fuweng11) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/administration/cluster_management.md]: Fixed.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/administration/cluster_management.md (hunk 1 above)
Comment (fuweng11) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/administration/cluster_management.md]: Fixed.

**Review by aloyszhang — APPROVED** [review]

**Review by justinwwhuang — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]

**Pull request merged**